service/controller/resource/tccp/create.go

```diff
@@ -143,6 +143,13 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		}
 	}
 
+	if key.IsNewCluster(cr) && cc.Status.TenantCluster.MasterInstance.EtcdVolumeSnapshotID == "" {
```

Comment:
adding also check for snapshotID in controller context so we avoid infinite spam of message that we dont need to create etcd snapshot

Reply:
We need a proper todo issue to drop this when the migration code is not relevant anymore. Would also be cool to have some standard logging here like we usually do for actions so the information are available once.

Reply:
done

issue created as well, just not sure that its on the right board

Reply:
Then put it in the team board.